src/modules/mod.rs

```diff
@@ -55,65 +55,73 @@ mod battery;
 use crate::config::RootModuleConfig;
 use crate::context::{Context, Shell};
 use crate::module::Module;
+use std::time::Instant;
 
 pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
```

Comment:
This should return `Module<'a>` instead now

Reply:
I tried and it's a bit over my head right now (this is my first rust project): it breaks all over the place as this has than a different return type as the custom module which is handled special. Will try again if you insist but would appreciate a merge without that change if possible...